File: chihun-jang/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    answer = [i for i in arr if i%divisor==0]
    if len(answer) ==0:
        answer.append(-1)
    return sorted(answer)

def solution2(arr, divisor):
    answer = [i for i in arr if i%divisor==0]
    if not answer:
        return [-1]
    return sorted(answer)

File: chihun-jang/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
  var answer = [];
  arr.forEach((ele) => {
    if (ele % divisor === 0) {
      answer.push(ele);
    }
  });
  if (answer.length === 0) {
    return [-1];
  }
  return answer.sort((a, b) => a - b);
}
